docs/content/en/blog/articles/dev-config/index.md

```diff
@@ -0,0 +1,321 @@
+
+If we want to run K8ssandra with limited resources, we’ll need to set these appropriately in our Helm values files.
+
+## Setting heap sizes in K8ssandra
+The K8ssandra Helm charts allow us to set heap sizes for both the Cassandra and Stargate pods separately.
+
+### Cassandra
+For Cassandra, the heap and new gen sizes can be set at the cluster level, or at the datacenter level (K8ssandra will support multi DC deployments in a future release):
+
+```
+cassandra:
+  version: "3.11.10"
+  ... 
+  ...
+  # Cluster level heap settings
+  heap: {}
+   #size:
+   #newGenSize:
+
+  datacenters:
+  - name: dc1
+    size: 3
+    ... 
+    ... 
+    # Datacenter level heap settings
+    heap: {}
+      #size:
+      #newGenSize:
+```
+
+By default, these values aren’t set, which lets Cassandra perform its own computations based on the available RAM, applying the following formula: 
+
+```
+max(min(1/2 ram, 1024MB), min(1/4 ram, 8GB))
+```
+
+The catch when you run several Cassandra nodes on the same machine is that they will all see the same total available RAM but won’t be aware that other Cassandra nodes could be running as well.
+When allocating 8GB RAM to Docker, each Cassandra node will compute a 2GB heap. With a 3 nodes cluster, it’s already 6GB of RAM used, not accounting for the additional off heap memory that can be used by each JVM.
+That doesn’t leave much RAM for the other components K8ssandra includes, such as Grafana, Prometheus and Stargate.
+
+The takeaway here: **leaving heap settings blank is not a good idea for a dev environment** in particular, where several Cassandra instances will be collocated on the same host machine. (By default, K8ssandra does not allow multiple Cassandra nodes on the same Kubernetes worker node. For this post, we’re using kind to have multiple worker nodes run on the same OS instance - or virtual machine in the case of Docker Desktop).
+
+The chosen heap size will directly impact the throughput you can expect to achieve (although it’s not the only limiting factor). A small heap will involve more garbage collections, which will generate more stop the world pauses and directly impact throughput and latency. It also increases the odds of running out of memory if the workload is too heavy, as objects cannot end their lifecycle fast enough for the available heap space.
+
+Setting the heap size at 500MB with 200MB of new gen globally for the cluster would be done as follows:
+
+
+```
+cassandra:
+  version: "3.11.10"
+  ... 
+  ...
+  # Cluster level heap settings
+  heap: 
+   size: 500M
+   newGenSize: 200M
+
+  datacenters:
+  - name: dc1
+    size: 3
+```
+
+### Stargate 
+Because Stargate nodes are special coordinator-only Cassandra nodes and run in the JVM, it is also necessary to set their max heap size:
```

Comment:
```suggestion
Because Stargate nodes contain special coordinator-only Cassandra nodes and run in the JVM, it is also necessary to set their max heap size:
```

Given there are the HTTP components as well I feel `contain` is more appropriate. WDYT?

Reply:
agree, will change

Reply:
actually I think original phrasing is still ok.